docker-compose.yml

```diff
@@ -210,7 +210,7 @@ services:
     command:
       >
       /bin/bash -c "wait-for.sh -t 0 -q elasticsearch:9200 --
-      wait-for.sh -t 0 -q services:1111 --
+      wait-for.sh -t 0 -q services:8545 --
```

Comment:
@mikeshultz have changed this back from port 1111 to 8454. It was preventing some of the services in docker to start up.

Reply:
Perhaps because something with services isn't working?  This is preventing other services from coming up because services (probably) isn't coming up.  It specifically brings up an empty service on port `1111` to indicate that everything is good to go:

https://github.com/OriginProtocol/origin/blob/2334d4e15a8026671445a9cf1587428ce284ae7d/packages/services/index.js#L461

By checking 8545(ganache), you're skipping basically every other service that it brings up.  I'd recommend looking into the cause of your issue instead of making this change.

Reply:
Ah thanks @mikeshultz The problem was with identity contract not deploying. Fix here: 
https://github.com/OriginProtocol/origin/pull/4274